sympy/calculus/euler.py

```diff
@@ -3,7 +3,7 @@
     combinations_with_replacement, iterable, range)
 
 
-def euler_equations(L, funcs=(), vars=()):
+def euler_equations(L, funcs=(), variables=()):
```

Comment:
If this is a public function, which it is, and you change the call signature, then it needs to be deprecated properly.

https://github.com/sympy/sympy/wiki/Deprecating-policy